.github/workflows/build-scan.yml

```diff
@@ -0,0 +1,19 @@
+name: build-scan
+on: 
+  # Since branch protection prevents pushing to development, pushes only happen when merging PRs, so this runs only on merges (there's no way to set GitHub Actions to run on merges)
+  push:
+    branches:
+      - development
+
+jobs:
+  build-scan:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: redhat-actions/buildah-build@v2
+      with:
+        image: dm-app-image
+        containerfiles: |
+          ./src/front-end/Dockerfile
```

Comment:
Is the omission of the postgres container intentional? Had using docker compose been considered as an alternate way to build both images? Replacing lines 13-17 with the following, for instance.
```
- name: build
run: docker-compose up
```

Reply:
I used buildah instead of docker/docker-compose up because docker's not used on openshift and I was aiming for parity. I left out the postgres container because 1) it's not used in prod, 2) it's the same container that's already on Docker Hub.

Reply:
Any benefit to avoiding the assumption that images on Docker Hub are free from vulnerabilities?

Reply:
>  it's the same container that's already on Docker Hub

Careful with that claim, while technically true our `./src/front-end/Dockerfile` here could be updated to [follow best practices](https://cheatsheetseries.owasp.org/cheatsheets/NodeJS_Docker_Cheat_Sheet.html), as [it currently tracks a tag](https://github.com/bcgov/digital_marketplace/blob/084e25444a5c008e807aab9339e70cb42802c54a/src/front-end/Dockerfile#L1) and not a `sha256` hash. As a result there's no guarantee that the docker image we pulled from docker hub using this tag is the same docker image that will be deployed when we rebuild on CI using this tag ([determinism](https://cheatsheetseries.owasp.org/cheatsheets/NodeJS_Docker_Cheat_Sheet.html#1-use-explicit-and-deterministic-docker-base-image-tags)).

Reply:
> Any benefit to avoiding the assumption that images on Docker Hub are free from vulnerabilities?

Supporting this: what if we assume that all images on Docker Hub or the [Red Hat Container Image Registry](https://catalog.redhat.com/software/containers/search) are frequently compromised?